### DISCUSSION TITLE
power pack

### DISCUSSION THREAD

**Discussion by AlanAliberti** [bhoogter/VB6TocSharp#8] [Q&A]
Hello, thanks for that code. 

The link to "visual basic power packs" seems to be dead and I just don't manage to find it elsewhere. Any known solution? Do you know about somewhere where I could pick the c# code for Val, Trim, etc?

Thanks.

Alan

**Comment by bhoogter** [comment]
I had feared that might eventually happen so I had added the setup file in the repo itself about 3 wks ago.

This should be what you need..

https://github.com/bhoogter/VB6TocSharp/blob/cbbaf90f8eb2e627fe844affe747760fa8807c3b/VisualBasicPowerPacksSetup.exe

**Comment by AlanAliberti** [comment]
Thanks for that, I hadn't noticed the file was there already, I've
installed it.

Sorry to be a pain but I'm still struggling with that... I'm
definitely missing something. I'm using Visual Studio 2019 and can't see a
way to add/use this reference in it. Would you mind pointing me to some
resource on that?

Thanks again for all your help.

**Reply by bhoogter** [reply]
Delayed response, but I try to get to them eventually... 

If you're still struggling with it, would a resource like this work:

> https://social.msdn.microsoft.com/Forums/en-US/a80970fd-8fc3-4154-b9ba-c635f0072246/how-do-i-download-and-install-powerpacks-for-visual-studio-express-comunity-2015?forum=vstest

Once the powerpack is installed, it should show up in the list.  While the download might be different, the process for adding/removing references should be the same.
